pb/pb_util.go

```diff
@@ -0,0 +1,122 @@
+package pb
+
+import (
+	"fmt"
+
+	"github.com/arr-ai/arrai/rel"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protodesc"
+	pr "google.golang.org/protobuf/reflect/protoreflect"
```

Comment:
Why the alias?

Reply:
It is shorter and simpler than `protoreflect `

Reply:
It doesn't occur often enough for the indirection to be warranted. I had to lookup what `pr` meant, where `protoreflect` would be obvious.